mikeio/spatial/_FM_utils.py

```diff
@@ -643,6 +644,16 @@ def _to_polygons(node_coordinates: np.ndarray, element_table: np.ndarray) -> lis
     return polygons
 
 
+def _create_node_element_matrix(element_table, num_nodes):
```

Comment:
I added the type hints. The type of the output is a csr_matrix actually. I also added a docstring for the function.

Reply:
Great, next thing to pay attention to is that we use [ruff](https://astral.sh/ruff) to format and check the code. If you use VS Code you can use the [Ruff extension](https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff)

We have some picky rules related to the docstrings. But it should be apparent in the editor, since the rules are defined in the pyproject.toml.

This is why the check is failing.
```
mikeio/spatial/_FM_utils.py:658:5: D413 [*] Missing blank line after last section ("Returns")
```

Reply:
Sorry I missed that formatting part. I used ruff extension in VS Code to fix the formats of the code and committed the changes